go/enclave/rpc/GetCustomQuery.go

```diff
@@ -19,29 +19,30 @@ func GetCustomQueryValidate(reqParams []any, builder *CallBuilder[common.Private
 		builder.Err = fmt.Errorf("unable to extract query - %w", err)
 		return nil
 	}
-	builder.From = &privateCustomQuery.Address
+	addr := privateCustomQuery.Address
+	builder.From = &addr
 	builder.Param = privateCustomQuery
 	return nil
 }
 
-func GetCustomQueryExecute(builder *CallBuilder[common.PrivateCustomQueryListTransactions, common.PrivateQueryResponse], rpc *EncryptionManager) error {
+func GetCustomQueryExecute(builder *CallBuilder[common.ListPrivateTransactionsQueryParams, common.PrivateTransactionsQueryResponse], rpc *EncryptionManager) error {
 	err := authenticateFrom(builder.VK, builder.From)
 	if err != nil {
 		builder.Err = err
 		return nil //nolint:nilerr
 	}
-
-	encryptReceipts, err := rpc.storage.GetTransactionsPerAddress(builder.ctx, &builder.Param.Address, &builder.Param.Pagination)
+	addr := builder.Param.Address
```

Comment:
what's the difference between this and `privateCustomQuery.Address`?

Reply:
Nothing, but we don't have a privateCustomQuery in this `Execute` method, privateCustomQuery got set on the builder as `builder.Param` in the `Validate` method.

Yeah this feels a little weird, but will tidy it up when we get rid of 'CustomQuery' from enclave, it will just be a 'GetPersonalTransactions' method and won't be trying to do anything generic.

Reply:
then let's add a check that they are equal. To avoid any security hole